ballerina/Tests/mock_service.bal

```diff
@@ -0,0 +1,60 @@
,109,102,134,122,204,4,9,16,144,25,9,113,132,29,155,17,185,144,18,133,193,17,17,43,222,150,160,240,175,213,0,97,197,1,192,32,247,69,248,143,57,88,184,75,186,2,75,230,77,90,26,107,84,171,153,223,20,29,50,67,83,82,99,49,52,99,87,118,55,119,195,24,79,51,228,211,56,93,49,144,65,128,211,6,5,11,128,24,168,18,94,38,18,134,129,131,145,241,56,81,153,61,16,0,76,20,22,0,4,2,39,18,70,149,131,170,139,237,39,134,159,229,205,18,141,56,80,213,12,25,1,44,50,39,38,138,116,128,70,152,96,136,12,80,97,89,139,156,5,2,2,1,152,194,9,147,16,152,137,81,146,132,5,70,148,64,44,6,6,12,108,197,182,67,97,24,66,152,172,212,212,42,129,164,74,167,101,139,69,116,33,49,21,64,64,172,213,179,198,19,172,20,2,208,75,142,208,210,13,213,105,237,5,134,180,165,76,235,83,64,44,230,85,156,50,233,49,39,234,84,206,161,249,84,52,202,156,25,2,55,42,64,65,195,33,45,78,5,47,57,212,164,223,139,179,54,6,76,38,50,3,21,140,68,91,51,194,4,200,161,195,2,13,76,152,89,50,241,124,203,4,179,25,4,65,162,195,42,156,12,246,118,49,176,68,193,35,3,31,143,12,98,28,38,13,152,16,34,97,0,58,212,2,129,144,117,212,6,136,50,232,141,90,163,74,72,66,36,214,71,58,19,193,206,140,114,179,100,188,209,17,74,22,36,56,12,204,43,54,76,66,15,0,9,153,130,230,48,11,80,97,204,252,192,13,50,68,84,40,8,44,201,21,50,67,64,195,90,138,55,24,48,198,36,35,21,47,112,8,18,27,67,101,157,45,107,200,166,79,204,213,59,41,47,232,0,25,134,22,99,7,152,193,46,179,33,46,9,128,2,6,2,175,81,8,196,146,50,164,146,4,255,243,228,196,190,123,76,57,76,51,220,208,0,42,0,20,25,173,150,117,10,68,35,13,27,67,118,216,49,9,133,52,105,81,25,145,136,212,202,94,64,168,35,18,33,79,3,65,24,80,72,109,47,135,87,209,129,20,102,204,33,104,20,65,141,20,14,12,253,174,215,240,26,8,195,134,87,168,220,89,34,227,41,180,158,25,103,45,118,157,253,136,33,144,4,1,104,154,248,32,57,138,22,164,130,192,76,48,147,16,28,12,21,126,161,56,194,140,49,162,148,136,92,1,137,20,99,69,0,133,44,228,65,48,195,76,112,212,250,64,73,128,2,130,176,117,166,66,4,6,97,131,163,243,210,160,42,10,177,86,21,47,75,186,128,87,25,186,162,105,101,75,106,189,83,216,194,140,50,35,146,229,1,69,174,117,212,197,180,2,146,52,172,141,42,32,113,17,24,67,26,64,200,140,1,14,99,200,9,48,195,76,96,180,114,85,82,228,163,235,243,85,144,151,4,0,5,77,162,108,53,97,85,42,165,80,20,133,80,88,218,152,169,172,81,115,38,51,204,165,37,225,129,209,229,58,96,4,19,32,9,199,6,5,49,2,86,113,123,75,138,242,37,242,63,51,243,4,52,200,17,66,161,16,19,16,52,198,13,49,0,86,232,84,49,149,60,104,207,15,1,0,140,49,226,18,9,27,80,212,84,65,159,96,110,90,85,76,65,77,69,51,46,49,48,48,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85,85];           
```

Comment:
Shall we defined the mock-payload as a constant and use it when returning the response ?